docs/CONFIG-PROPERTIES.md

```diff
@@ -10,6 +10,7 @@
 | timer.location.cloud.interval | integer in seconds | 1 hour | how frequently device reports geographic location information to controller |
 | timer.location.app.interval | integer in seconds | 20 | how frequently device reports geographic location information to applications (to local profile server and to other apps via meta-data server) |
 | timer.send.timeout | timer in seconds | 120 | time for each http/send |
+| timer.dial.timeout | timer in seconds | 10 | maximum time allowed to establish connection |
```

Comment:
FWIW I was concerned that this might only be one retransmit of the SYN (because I tought the Linux SYN timeout was 4 seconds with binary exponential backoff) but a quick test is telling me it is 1 second, meaning that we get 3 retransmitted SYNs in 10 seconds which should be sufficiently robust against random packet drops.